core/node/da_clients/Cargo.toml

```diff
@@ -12,6 +12,7 @@ categories.workspace = true
 
 [dependencies]
 serde = { workspace = true, features = ["derive"] }
+eq-common = { path = "../../../../eq-service/common", features = ["grpc"] }
```

Comment:
```suggestion
```

This _should_ be re-exported in `eq-sdk` and if it's not, let's make it so

Reply:
let's be sure to only use https://crates.io/crates/eq-sdk/0.1.1 if we can here, but common is published now too